node/_bin/munin-asyncd.in

```diff
@@ -154,8 +158,7 @@ MAIN: while($keepgoing) {
 		}
 
 		# Should update it
-		$last_updated{$plugin} = $when;
-		$when_next = min($when_next, $when + max($plugin_rate, $minrate));
+		$last_updated{$plugin} = $when - 180;
```

Comment:
Should this be `$when - 120` instead?
I am confused, that we need to infer this timestamp indirectly. Is there no better way?
If we really need to do this, then we should define a constant somewhere (e.g. `0.75 * $minrate`) instead of using two hard-coded numbers in different locations.

Reply:
Again I have to say that I can't remember anymore. There was some reason, so I'll have try it without first, and then maybe define constant. I assume back in 2.0.18 days the $when / $when_next logic was different than currently, and this part is not needed anymore.